website/templates/profile/account.mako

```diff
@@ -66,12 +65,15 @@
                                 <tr>
                                     <td colspan="3">
                                         <form data-bind="submit: addEmail">
+                                        <p>If you have more than one OSF account and would like to merge them or would like to be able to log in under multiple email addresses, add your secondary email address in the field below.</p>
```

Comment:
Two suggestions- pinging original ticket author @jlcohoon for language feedback.

1. Users can have more than one alternate email address; change "secondary" to "alternate" (this is also more consistent with the UI above)
2. Because this box is doing two separate (but related) things (one of them very powerful), perhaps we should split the description into separate lines?

For example,

```
<p>
    If you would like to log in using more than one email address, enter your alternate address in the field below.
</p>
<p>
    If the email address you enter is already registered to another OSF user, an email will be sent to the new address with a confirmation link to merge that account into yours.
</p>
```